geometry/target_config_old.yaml

```diff
@@ -0,0 +1,47 @@
+targetOpt: 18 #equal to the number of sublayers of target
+#    if targetOpt < 10:
+#     c.target.sl            =  1*u.cm  # air slit total length
+#     c.target.length        =  50*u.cm + c.target.sl * (targetOpt-1)
+#          material,length
+target:
+    z: 0
+    z0: 0
+    Nplates: 10 #number of sublayer blocks
+    M:
+        - "molybdenum"
+        - "molybdenum"
+        - "molybdenum"
+        - "molybdenum"
+        - "molybdenum"
+        - "tungsten"
+        - "tungsten"
+        - "tungsten"
+        - "tungsten"
+        - "tungsten"
+
+    L:
+        - 8. #*u.cm
+        - 2.5 #*u.cm
+        - 5.0 #*u.cm
+        - 6.5 #*u.cm
+        - 8. #*u.cm
+        - 5. #*u.cm
+        - 8. #*u.cm
+        - 10. #*u.cm
+        - 20. #*u.cm
+        - 35. #*u.cm
+
+    N:
+        - 1
+        - 7
+        - 2
+        - 1
+        - 2
+        - 1
+        - 1
+        - 1
+        - 1
+        - 1
+
+    sl:  0.5  # [cm] H20 slit *17 times
```

Comment:
The assumption that all the slits have the same size does not appear to be true anymore with the new target.